.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "Adafruit_Python_GPIO"]
 	path = Adafruit_Python_GPIO
 	url = https://github.com/aninternetof/Adafruit_Python_GPIO
+[submodule "GrovePi"]
+	path = GrovePi
+	url = https://github.com/rovercode/GrovePi.git 
+	branch = firmware-fixes-1.3.0
```

Comment:
This branch will always exist? Should we tag something here or merge it?

Reply:
I'll merge it.

Reply:
Done.